doc/Developing/os/Health-Information.md

```diff
@@ -31,6 +31,7 @@ Currently we have support for the following health metrics along with the values
 | temperature                     | C                           |
 | voltage                         | V                           |
 | waterflow                       | l/m                         |
+| versatile                       |                             |
```

Comment:
What's this new unit supposed to show?

Reply:
Just read in your comment. Not sure about this one. I can definitely see the benefit but how will you deal with things like min max scales etc?

I'd prefer to see one of the other PRs that adds custom OIDs in supported rather than go down this route.

Reply:
Unit would, if we push this concept further, be a parameter of this versatile type. And min/max would be defined as today in the YAML (but of course, auto-compute of the min/max value would not make sense here). 

In a PBX like here, creating a dedicated type for each entry would be too much (we can count PhonesSets, SIP subscriptions, etc etc). But we still need a way to show these. 

What do you mean by "one of the other PRs" ?

Reply:
Min / max I'm referring to scales rather than thresholds. I.e what the graphs have in the Y axis as the min/max.

8738 and 7925 both try and cover this in a more generic way.

Reply:
Ok.
May be I'll take another route right now, which is to do a standard polling instead of Yaml, with a dedicated page. This would allow this PR to continue its life without creating this graph type that would later on need to be changed/suppressed. 
What do you think ?

Reply:
If you want to get this merged, just omit that sensor for now and add it later.

Reply:
In the process of doing that. WIP will be removed when tests are done.